_data/sites.json

```
@@ -15975,6 +15975,14 @@
         ]
     },
 
+    {
+       "name": "Payback",
+        "url": "https://www.payback.de/info/mein-payback/kontakt",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
If it requires contacting, we put it as hard.